docs/ci-reference.md

```diff
@@ -29,3 +29,15 @@ If the need presents itself to run odo integration tests against a specific vers
         - oc login -u developer
         - make test-main-e2e
 ```
+
+# How to run integtration tests on Prow
```

Comment:
> @amitkrout Hmm. Not too sure where we can put the documentation. I think adding it to just the Makefile would suffice.

@cdrage Here we can add the doc for reference with some more details. PTAL

Reply:
Ah, I had forgotten about the ci-reference.md doc, yes, there is good.